c++/include/orc/Reader.hh

```diff
@@ -657,6 +657,12 @@ namespace orc {
      * @param rowNumber the next row the reader should return
      */
     virtual void seekToRow(uint64_t rowNumber) = 0;
+
+    /**
+     * Get the current stripe position entries for the specified column.
+     * @return the position entries for the specified column.
+     */
+    virtual std::vector<std::vector<int>> getCurrentStripePositionEntries(uint64_t columnId) = 0;
```

Comment:
nit: I was thinking if we should design a better data structure, something like:
```
struct RowGroupPositions {
  uint64_t columnId;
  std::vector<int32_t> positions;
}
```
Then it can be reused if we want to add `std::vector<RowGroupPositions> getPositionEntries(int stripe, std::vector<int> columnIds)` in the future.

Reply:
I'll mark it in the next work.

Reply:
I'd suggest to do this all at once. If we don't change it in time, the public API will get released and we cannot change it any more.